contracts/facets/ThirdPartyFacet.sol

```diff
@@ -29,11 +29,9 @@ contract ThirdPartyFacet {
     /// @param sarcoId The identifier of the sarcophagus to clean
     /// @param paymentAddress The address to which rewards will be sent
     function clean(bytes32 sarcoId, address paymentAddress) external {
-        LibTypes.Sarcophagus storage sarco = s.sarcophagi[sarcoId];
```

Comment:
Shoot, only just caught this while working on my own branch but we shouldn't be reverting on inactive sarcophagi here. I'll update.

Reply:
@katien -- doesn't the sarcophagus need to be active to clean it? Inactive states would be `Buried`, `Cleaned` or `Accused` -- and don't think any of those states could be cleaned.